external/buildscripts/build_classlibs_osx.pl

```diff
@@ -36,5 +36,6 @@
 	"--artifact=1",
 	"--artifactscommon=1",
 	"--buildusandboo=1",
+	"--buildMSBuild=1",

```

Comment:
Should we add this same line to build_classlibs_wsl.pl in case someone wants to build on windows?